### PR TITLE
DSERVE-81 new model

### PR DESCRIPTION
**Pull request by mingjiecn**

### CODE REVIEW

File: genomic_data_service/regulome_atlas.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 try:
     TRAINED_REG_MODEL = pickle.load(
-        open('./ml_models/rf_model.sav', 'rb')
+        open('./ml_models/rf_model1.0.1.sav', 'rb')
```
Comment (ottojolanki) [inline genomic_data_service/regulome_atlas.py:19]: We should also update the model in s3, as well as the code that fetches the models.

**Review by ottojolanki — CHANGES_REQUESTED** [review]
Need to upload the new model file to s3, and update the code that fetches the model.

File: genomic_data_service/regulome_atlas.py (hunk 1 above)
Comment (mingjiecn) [reply genomic_data_service/regulome_atlas.py:19]: I already uploaded the model to s3 bucket and changed the code to fetch that model.

**Review by ottojolanki — APPROVED** [review]
LGTM

**Pull request merged**